python/lsst/obs/subaru/rings.py

```diff
@@ -134,10 +141,14 @@ def fitPatches(exp, nx=4, ny=8, bin=None, frame=None, returnResidualImage=False,
         assert ccd is not None, "I need a CCD to set r and the logarithmic gradient"
         assert r is not None and lnGrad is not None, "Please provide both r and lnGrad"
 
-    z = afwImage.ImageF(nx, ny)      za = z.getArray()
```

Comment:
rings.py had a bunch of stuff that made it clear it hasn't been run in a long time, if ever.